kep/625-node-resource-fit-plus-scoring/README.md

```diff
@@ -0,0 +1,114 @@
+# Node Resource Fit plus Scheduling
```

Comment:
This seems very similar to an old plugin. Can you help to tell the difference or integrate it?
FYI: https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/kep/48-node-resources-allocatable-scoring

Reply:
@googs1025 The old version of the plug-in cannot distinguish between different strategies for resource use and can only treat them equally.

Reply:
Older version policies can only use one policy for different resources. Not suitable for complex resource scenarios, such as AI

Reply:
@googs1025 Under the AI ​​cluster. It is hoped that GPU tasks will be clustered and CPU tasks will be scattered. The old version of the strategy is not supported.

Reply:
As I mentioned, it seems to be very similar to the previous nodeResourcesAllocatable, and I don't think it needs to be extended with a new plugin. If it is possible, can it be integrated into the original plugin? 🤔

Reply:
@googs1025 What you mean is that you agree with this strategy, but the implementation method is changed to modify the original nodeResourcesAllocatable strategy.

Reply:
> @googs1025 What you mean is that you agree with the design of the NodeResourcesFitPlus strategy, but you want to implement it by modifying the original nodeResourcesAllocatable strategy?

@googs1025 Do I understand correctly?